funnel/templates/project.html.jinja2

```diff
@@ -1,4 +1,5 @@
 {% extends "layout.html.jinja2" %}
+{% set title_suffix = project.profile.title %}
```

Comment:
This is not being used. Also, it's usually a prefix.

Reply:
We have defined it in layout
https://github.com/hasgeek/funnel/blob/master/funnel/templates/layout.html.jinja2#L5